pkg/controller/namespace/namespace_controller.go

```diff
@@ -205,6 +189,17 @@ func (r *ReconcileNamespace) bindWorkspace(namespace *corev1.Namespace) error {
 	return nil
 }
 
+// Reset the ownerReference of the namespace
```

Comment:
So this function actually removes Workspace from its ownerReference? I think we should choose another function name, reset usually means clear all.

Reply:
/cc @wansir

Reply:
done